github-for-jira.sd.yml

```diff
@@ -601,7 +601,7 @@ environmentOverrides:
             MetricName: ApproximateNumberOfMessagesVisible
             Namespace: AWS/SQS
             Description: "Message count for push queue too high! Follow the runbook: https://hello.atlassian.net/wiki/spaces/PF/pages/1550488861/HOWTO+Respond+to+Too+Many+Messages+on+the+Queue+Alert"
```

Comment:
on line 249 is the scaling rules expression. To save time and lives here it is

            `- Expression: "MAX([cpuUsage, memoryUsage, (pushQueueMessages/250)*100, 
            (backfillQueueMessages/20)*100, (branchQueueMessages/250)*100, 
            (deploymentQueueMessages/250)*100])"`